app/controllers/application_controller.rb

```diff
@@ -1,160 +1,143 @@
+# Superclass for all model controllers
 class ApplicationController < ActionController::Base
   protect_from_forgery
 
   helper :html
   helper :my_content
-  
+
   helper_method :nl2br
-  
+
   helper_method :universe_filter
 
-  # Rails changed cookie format in rails 4, so log out all old users so they get the new version
+  # Rails changed cookie format in rails 4, so log out all old users so that
+  # they get the new version
   rescue_from JSON::ParserError, with: :force_user_logout
   def force_user_logout
     reset_session
     redirect_to login_path
   end
-  
+
   # View Helpers
   def nl2br(string)
-    #simple_format string
-    string.gsub("\n\r","<br>").gsub("\r", "").gsub("\n", "<br />").html_safe
+    # simple_format string
+    string.gsub('\n\r', '<br>').gsub('\r', '').gsub('\n', '<br />').html_safe
```

Comment:
Does this change actually break this method? I was under the impression that strings with escaped characters have to be double-quoted, otherwise you get the literal string `\n`.